src/main/java/frc/robot/Constants.java

```diff
@@ -131,6 +146,7 @@ public class Rotation {
     }
 
     public static final double k_autoAimDistanceThreshold = 1; // in meters
+    public static final double k_autoAimOmegaRPSThreshold = Math.PI / 8; // in radians
```

Comment:
Not a huge issue, but the unit in the comment is wrong; should be radians/second 🤓 ☝️